src/cli/bin.ts

```diff
@@ -56,16 +57,33 @@ yargs
         : undefined;
       const servers = await fs.readJSON(serversPath);
 
-      const migratedUIFolder = await migrateUIFolder(
+      const [migratedUIFolder, migrationReport] = await migrateUIFolder(
         legacyUIFolder,
         servers,
         keysOfWidgetPluginsToRemove,
         legacyPivotFolder,
       );
 
+      const { dir } = path.parse(outputPath);
+
       await fs.writeJSON(outputPath, migratedUIFolder, {
         spaces: 2,
       });
+
+      if (migrationReport) {
+        await fs.writeJSON(
+          path.join(...dir, "migration-report.json"),
+          migrationReport,
+          {
+            spaces: 2,
+          },
+        );
+        console.log(
```

Comment:
And error log would be more adapted here.

```suggestion
        console.error(
```